source/transactions/transactions.rst

```diff
@@ -220,14 +220,7 @@ is NULL or the readConcern in defaultTransactionOptions is NULL, the
 readConcern is inherited from the MongoClient associated with this
```

Comment:
Based on the above, we might copy the readConcern from the client, but only when starting the transaction.

When we run the first read operation, we'll use whatever read concern that was copied into the TransactionOptions.

This is the behavior we expect, right?

Reply:
That's right. I'll make that super-clear in " Constructing the first command within a transaction"